src/components/playground.tsx

```diff
@@ -1,28 +1,80 @@
-import { MidenContextProvider } from '@/lib/context-providers';
+'use client';
+import { useMiden } from '@/lib/context-providers';
 import { Header } from '@/components/header';
 import { ResizableHandle, ResizablePanel, ResizablePanelGroup } from '@/components/ui/resizable';
 import { MainWindow } from '@/components/main-window';
 import { Tabs } from '@/components/tabs';
 import { Inspector } from '@/components/inspector';
+import { useRef } from 'react';
+interface PanelHandle {
+	collapse: () => void;
+	expand: () => void;
+	getId: () => string;
+	getSize: () => number;
+	isCollapsed: () => boolean;
+	isExpanded: () => boolean;
+	resize: (size: number) => void;
+}
 
 export function Playground() {
+	const inspectorPanelRef = useRef<PanelHandle>(null);
+	const { isCollapsedTabs, collapsedTabs } = useMiden();
```

Comment:
Use present simple for action names

```suggestion
	const { isCollapsedTabs, collapseTabs } = useMiden();
```